pkg/serverless/invocationlifecycle/trace_test.go

```diff
@@ -7,27 +7,48 @@ package invocationlifecycle
 
 import (
 	"os"
+	"strconv"
 	"testing"
 	"time"
 
 	"github.com/DataDog/datadog-agent/pkg/trace/api"
 	"github.com/stretchr/testify/assert"
 )
 
-func TestStartExecutionSpan(t *testing.T) {
-	startTime := time.Now()
-	startExecutionSpan(startTime)
+var testString = `a5a{"resource":"/users/create","path":"/users/create","httpMethod":"GET","headers":{"Accept":"*/*","Accept-Encoding":"gzip","x-datadog-parent-id":"1480558859903409531","x-datadog-sampling-priority":"1","x-datadog-trace-id":"5736943178450432258"}}0`
```

Comment:
It looks like for some of these tests, you defined the test strings within the tests, but for other tests, you defined the test strings at the top of the file. I would move them all within the tests for improved consistency.